notes/style-guide.md

```diff
@@ -4,31 +4,44 @@ Style guide for the standard library
 This is very much a work-in-progress and is not exhaustive.
 
 ## File structure
+Agda programs are structured in modules. The first module in each file of the library is the top-level module whose name always matches the filename.
 
-#### Module imports
+#### Module import guidelines
 
 * All module imports should be placed at the top of the file immediately
   after the module declaration.
 
-* If the module takes parameters that require imports from other files
+* If the module takes parameters that require imports from other files,
   then those imports only may be placed above the module declaration.
 
 * If it is important that certain names only come into scope later in
   the file then the module should still be imported at the top of the
-  file but it can be given a shorter name using `as` and then opened
-  later on in the file when needed, e.g.
+  file but it can be given a shorter name using the keyword `as` and then opened later on in the file when needed, e.g.
```

Comment:
I'd probably prefer that you didn't get rid of the newlines as I was trying to stick to the standard 60 character limit here. I have a relatively small monitor and it makes these files very difficult to read if the lines aren't split.

Reply:
Sorry, I didn't realized that, maybe we should include this as a guideline also :-)

Reply:
Yup! Just added it to #1018

Reply:
But i think you meant 80, right? I tried soft wrap 60 and is cutting the text.